_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Quarkiverse Hub - Home of Community Extensions
+date: 2020-11-17
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
+
+Some have since then created their own repositories external to Quarkus, but others just never happened because not everyone wants the overhead of maintaining a build setup. Even the Quarkus core developers sometimes had ideas for an extension but not solid enough to be included in the Quarkus core repository.
+
+Enter the *Quarkiverse*!
+
+image:/assets/images/posts/quarkiverse/quarkiverse.png[]
+
+Quarkiverse is everything that encompasses the Quarkus universe: that includes the core, platform and all extensions.
+
+When it comes to developing extensions maintained by the community, it is important to standardize the repository structure, the release process and provide an integration with the existing Quarkus tooling.
+
+That's where Quarkiverse Hub comes into play.
+
+== What is Quarkiverse Hub?
+
+The link:https://github.com/quarkiverse[Quarkiverse Hub] is our proposed solution to create a "home" for Quarkus extensions maintained by the community.
```

Comment:
partial screenshot of quarkiverse hub front page with list of extensions would be nice here